bundles/automation/org.eclipse.smarthome.automation.module.core/ESH-INF/automation/moduletypes/ItemTriggers.json

```diff
@@ -128,6 +138,40 @@
                "required":true
             },
             {  
+               "name":"oldState",
```

Comment:
@kaikreuzer: I added it for uid: "core.ItemStateChangeTrigger", so yes it is the ItemStateChange-Event